bin/README.md

```diff
@@ -0,0 +1,26 @@
+## Bin
+
+This directory contains a number of scripts for running pathway parameter advising, updating reference pathways, and installing PGD.
+
+Most of these scripts must be run inside the `bin` directory. 
```

Comment:
Which ones do not need to be run from `bin`?

Reply:
Changed language to reflect that all now need to be run from bin.